docs/status.md

```diff
@@ -209,6 +211,16 @@ to CrateDB.
 </td>
 </tr>
 
+<tr>
+<td>
+<b>Documentation</b>
+</td>
+<td>
+<a href="https://github.com/crate/about/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/about/tests.yml?branch=main&label=About CrateDB" loading="lazy"></a>
+</td>
+</tr>
```

Comment:
@bmunkholm: The only leftover item is not exclusively a documenation artefact, as it also includes code.